bin/sync.sh

```diff
@@ -3,27 +3,22 @@
 # `-f` escapes `*`
 set -xf
 
-echo "Before Syncs"
-echo $(date +%s)
+BUCKET_NAME=${BUCKET_PATH:5}
 
 # Sync static assets first, before uploading other files that reference them.
-# Set long live cache headers.
-aws s3 sync ./_site ${BUCKET_PATH} \
-    --exclude "*" \
-    $(jq .paths[] static/staticfiles.json | xargs -I {} echo -n "--include static/{}* ") \
-    --cache-control "max-age=315360000,public,immutable" \
-    --acl public-read \
-    --delete
+# Set long live cache headers using `aws s3api` only to files that changed.
+./bin/rclone --config ./bin/rclone.conf --no-update-modtime -v \
+       $(jq .paths[] static/staticfiles.json | xargs -I {} echo -n "--include static/{} ") \
+       sync ./_site ${BUCKET_PATH} 2>&1 | grep "Copied" | cut -d ":" -f 4 | xargs -I '{}' aws s3api copy-object \
+                                                                        --acl public-read \
+                                                                        --copy-source ${BUCKET_NAME}/'{}' \
+                                                                        --bucket  ${BUCKET_NAME} \
+                                                                        --key '{}' \
+                                                                        --metadata-directive REPLACE \
+                                                                        --cache-control "max-age=315360000,public,immutable"
 
-echo "After Static Sync, before Non-static Sync"
-echo $(date +%s)
 
 # Sync the rest of the files.
-aws s3 sync ./_site ${BUCKET_PATH} \
-    --include "*" \
-    $(jq .paths[] static/staticfiles.json | xargs -I {} echo -n "--exclude static/{}* ") \
-    --acl public-read \
-    --delete
-
-echo "After Static Syncs"
-echo $(date +%s)
+./bin/rclone --config ./bin/rclone.conf --no-update-modtime \
```

Comment:
Do we want the no-update-modtime here?  It seems good for the previous command because we're uploading and then modifying the files. It probably doesn't matter too much either way, but might be keep the list of last time we touched a file up to date and true in s3.

Reply:
Good point. I'll remove this.